projects/wpdx/src/app/rehab-prio/rehab-prio.component.ts

```diff
@@ -737,6 +738,18 @@ export class RehabPrioComponent implements OnInit {
         this.rpState.map.setLayoutProperty(layer.id, 'visibility', this.rpState.show_landcover ? 'visible' : 'none');
       }
     });
+
+    if (this.rpState.show_adm_borders){
+
+      this.rpState.map.setFilter('adm-analysis-borders', null);
+      this.rpState.map.setPaintProperty('adm-analysis-borders', 'line-color','#5D3FD3');
+      this.rpState.map.setPaintProperty('adm-analysis-borders', 'line-opacity', 0.2);
+      this.rpState.map.setLayoutProperty('adm-analysis-borders', 'visibility', 'visible');
```

Comment:
you also need to turn them off if show_adm_borders is false and we're not in adman/staleness modes.